apps/mark/frontend/src/pages/unconfigured_screen.tsx

```diff
@@ -12,14 +12,14 @@ export function UnconfiguredScreen({
   return (
     <Screen>
       <Main centerChild>
-        <Prose textCenter>
-          <h1>VxMark is Not Configured</h1>
+        <CenteredLargeProse>
+          <h1>VxMark is not configured</h1>
```

Comment:
I was thinking maybe we change all the apps to use title case "<app> is Not Configured" to be more consistent with how other full-screen messages are capitalized

Reply:
Updated in [fc5ae77](https://github.com/votingworks/vxsuite/pull/3332/commits/fc5ae7797c5d4aa49ed29075105b1658bfa76c86)